src/app/admin/createCollectionCentre/CreateCollectionCentreForm.tsx

```diff
@@ -68,12 +78,14 @@ const CreateCollectionCentreForm: React.FC<{}> = () => {
             setSubmitDisabled(false);
 
             const logId = await logErrorReturnLogId("Error with insert: collection centre", error);
```

Comment:
appreciate it's not your change...
```suggestion
            const logId = await logErrorReturnLogId("Error with insert: collection centre", { error });
```